product_variant_configurator/models/product_template.py

```diff
@@ -44,13 +44,6 @@ def onchange_no_create_variants(self):
                 }
             }
 
-    @api.model
-    def create(self, vals):
```

Comment:
Why are you removing this?

Reply:
It seemed deprecated; we supply product_name in the context within the create method in product_product, but how is this supposed to influence the name in product_template? I could not find mention of it elsewhere, but I can look into it.

Reply:
Then don't remove the test that is checking this, and specify in the docstring of the test that is for ensuring the correct behavior, which was not correct in the past.

Reply:
And please do this in other commit, as it's something unrelated.

Reply:
I've made another commit with the changes to the restored unit test (or did you mean another pull request?).